packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1941,8 +1941,14 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   @protected
   bool hitTestChildren(BoxHitTestResult result, { required Offset position }) {
     final Offset effectivePosition = position - _paintOffset;
-    final InlineSpan? textSpan = _textPainter.text;
-    switch (textSpan?.getSpanForPosition(_textPainter.getPositionForOffset(effectivePosition))) {
+    final GlyphInfo? glyph = _textPainter.getClosestGlyphForOffset(effectivePosition);
+    // This works even with large letter-spacing and text justification, as
```

Comment:
reading this i am not sure what `works` mean. Do you mean it will missed when click in between character? or it would still hit the span. I think it is the latter, but would be good to be more clear

Reply:
It would still hit the span (same behavior as `getSpanForPosition` ). I'll update the comment.